openoa/analysis/aep.py

```diff
@@ -1003,7 +1003,12 @@ def run_AEP_monte_carlo(self):
             iav[n] = gross_lt_annual.std() / gross_lt_annual.mean()
             avail_pct[n] = avail_lt_losses
             curt_pct[n] = curt_lt_losses
-            lt_por_ratio[n] = (gross_lt.sum() / self._run.num_years_windiness) / gross_por.sum()
+            gps = (
+                gross_por.sum()
+                if not isinstance(gross_por, (pd.Series, pd.DataFrame))
+                else gross_por.values.sum()
+            )
+            lt_por_ratio[n] = (gross_lt.sum() / self._run.num_years_windiness) / gps
```

Comment:
What is the purpose of this code change? This is not in develop_v3. Just checking to make sure it's not a mistake.

Reply:
This has to do with how we single value Series can be interacted with vs numpy arrays. Essentially in a future version of pandas the original version will just break, which makes for a really annoying warning coming out of pandas.